### DISCUSSION TITLE
Advanced import setting save to file default path

### DISCUSSION THREAD

**Discussion by SmartySmart702** [godotengine/godot-proposals#6014] [3D]
### Describe the project you are working on

3D game with imported models

### Describe the problem or limitation you are having in your project

I like to use meshes directly instead of the glb scene so I use the "save to file" function. Right now I need to set the path manually for each mesh.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

There should be a default path to the current directory of the file (name would be the name of the object).

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

there is a default path and name but of course you can change it if you want to.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Its just a quality of life improfement that can save some time

### Is there a reason why this should be core and not an add-on in the asset library?

Because its a timesaver

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

___

There's an **Actions…** menu in the top-left corner of the Advanced Import Settings dialog, have you tried it?

![image](https://user-images.githubusercontent.com/180032/209696012-54b92a8d-1775-41af-a06a-1b3a249a15ed.png)

**Comment by SmartySmart702** [comment]
OMG sorry, I'm blind :D Now I can even group models in one file and import/update all at once.

**Reply by Calinou** [reply]
To be fair, that menu button's visibility could be improved – it's not easy to spot. I'm not sure how exactly it could be made more visible or easier to discover though.
